### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: src/docs/welcome/environments/DOC.md [hunk 1]
```diff
@@ -15,10 +15,11 @@
...
+* https://cloud.redhat.com/
+* https://qaprodauth.redhat.com/{beta}
 
 ### Uses QA auth
 
-* https://ci.cloud.paas.psi.redhat.com/
-* https://qa.cloud.paas.psi.redhat.com/
-* https://nightly.cloud.paas.psi.redhat.com/
+* https://ci.cloud.redhat.com/{beta}
+* https://qa.cloud.redhat.com/{beta}
```
Comment (kruai) [inline src/docs/welcome/environments/DOC.md]: What determines whether you include `{beta}` for this section?

File: src/docs/repos/deployment/DOC.md [hunk 2]
```diff
@@ -18,21 +18,23 @@ Should build to the [build repo](https://github.com/RedHatInsights/insights-depl
...
     * Verify that you are inside of your development repo and then encrypt the key with Travis CLI
-6. You should now have a deploy_key.enc file. Delete deploy_key and deploy_key.pub.
-7. Replace the deploy_key decryption statement in .travis.yml with your newly generated one from the encryption step 5.b but replace ~\/.ssh/deploy_key with just deploy_key
-8. Commit your deploy_key.enc file, .travis.yml, and .travis directory
-9. Verify that the Travis env variables have been added.
-10. Confirm the build completed successfully on Travis-CI and verify that the files were pushed to your build repo.
+    * `travis encrypt-file deploy_key`
+        * do not add the `openssl` command to the `.travis` file.
+6. You should now have a `deploy_key.enc` file. Delete `deploy_key` and `deploy_key.pub`.
+7. Commit your deploy_key.enc file, .travis.yml, and .travis directory
```
Comment (kruai) [inline src/docs/repos/deployment/DOC.md]: If you put the file names in `code formatting` for step 6, I'd continue doing it here for consistency.

File: src/docs/welcome/platform/DOC.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+## Chrome
+
+Chrome (or Insights Chrome) is the "wrapper" around your application. At the moment, Chrome does a lot of thigs for the platform:
+
+- Serves the sidenav (generated by the source of truth) & routes accordingly - injected with ESI tags
+- Serves the masthead - injected with ESI tags
+- Authenticates with Keycloakjs
+- Window functions
+- Inventory Component
+- CSS Variables
```
Comment (kruai) [inline src/docs/welcome/platform/DOC.md]: The first 3 in this list describe functionality provided, but the last 3 are just nouns. These should probably be in the same format

File: src/docs/welcome/platform/DOC.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Examples:
+
+- [prod-stable](https://github.com/RedHatInsights/cloud-services-config/blob/prod-stable/main.yml)
+- [ci-beta](https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/main.yml)
+
+## Proxy
+
+The Proxy (or Insights Proxy) is a container that uses [Spandx](https://github.com/RedHatInsights/spandx) which provides ESI tags. This allows the sidenav and masthead from chrome to get injected and also allows routing between apps.
+
+Developers can pass custom spanx config files
```
Comment (kruai) [inline src/docs/welcome/platform/DOC.md]: should be Spandx

**Review by kruai — CHANGES_REQUESTED** [review]
Great work! I just have a few notes.

File: src/docs/welcome/platform/DOC.md (hunk 4 above)
Comment (ryelo) [reply src/docs/welcome/platform/DOC.md]: spanx lol

File: src/docs/deployments/overview/DOC.md [hunk 5]
```diff
@@ -23,14 +23,14 @@ Below is a list of the branches on each app build repo and the environments that
 
 * prod-stable: https://cloud.redhat.com/
 * prod-beta: https://cloud.redhat.com/beta/
-* qa-stable: https://qa.cloud.paas.psi.redhat.com/
-* qa-beta: https://qa.cloud.paas.psi.redhat.com/beta
-* ci-stable: https://ci.cloud.paas.psi.redhat.com/
-* ci-beta: https://ci.cloud.paas.psi.redhat.com/beta
+* qa-stable: https://qa.cloud.redhat.com/
+* qa-beta: https://qa.cloud.redhat.com/beta
+* ci-stable: https://ci.redhat.com/
```
Comment (yashnalla) [inline src/docs/deployments/overview/DOC.md]: this is not in the scope of the doc changes but why is ci-stable the only url that violates the pattern?

File: src/docs/repos/chrome/DOC.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# Insights Chrome
+# Chrome
```
Comment (yashnalla) [inline src/docs/repos/chrome/DOC.md:1]: you removed 'Insights' here but not in the README?

**Review by tahmidefaz — DISMISSED** [review]
Looks good! :+1: :+1:

File: src/docs/deployments/overview/DOC.md (hunk 5 above)
Comment (tahmidefaz) [reply src/docs/deployments/overview/DOC.md]: ^ same!

Comment (ryelo) [reply src/docs/deployments/overview/DOC.md]: twas a typo

Comment (ryelo) [reply src/docs/deployments/overview/DOC.md]: fixed!

**Review by ryandillinfelton — DISMISSED** [review]

**Review by kruai — DISMISSED** [review]
Looks great now!

**Review by tahmidefaz — DISMISSED** [review]

**Review by yashnalla — CHANGES_REQUESTED** [review]

File: src/docs/repos/chrome/DOC.md (hunk 6 above)
Comment (ryelo) [reply src/docs/repos/chrome/DOC.md:1]: Now it's gone 😄

**Review by kruai — APPROVED** [review]

**Review by yashnalla — APPROVED** [review]

**Pull request merged**